DTIPrep/DTIPrep_pipeline.pl

```diff
@@ -83,14 +83,29 @@
 
 
 
-# These settings are in a config file (profile)
-my  $data_dir       =   $Settings::data_dir;
-my  $t1_scan_type   =   $Settings::t1_scan_type;
-my  $DTI_volumes    =   $Settings::DTI_volumes;
-my  $reject_thresh  =   $Settings::reject_thresh;
-my  $niak_path      =   $Settings::niak_path;
-my  $QCed2_step     =   $Settings::QCed2_step;
-
+# These settings are in the ConfigSettings table
+my  $data_dir       =   NeuroDB::DBI::getConfigSetting(
+                        \$dbh,'dataDirBasepath'

```

Comment:
Global symbol "$dbh" requires explicit package name (did you forget to declare "my $dbh"?)

also 

Can't use 'defined(@array)' (Maybe you should just omit the defined()?) at ./DTIPrep_pipeline.pl line 67